DESCRIPTION

```diff
@@ -1,48 +1,26 @@
 Package: hubEnsembles
-Title: Ensemble methods for combining hub model outputs
+Title: Ensemble Methods for Combining Hub Model Outputs
 Version: 0.1.6
 Authors@R: c(
-    person(given = "Evan L",
-           family = "Ray",
-           email = "elray@umass.edu",
-           role = c("aut")),
-    person(given = "Li",
-           family = "Shandross",
-           role = c("aut", "cre"),
-           email = "lshandross@umass.edu",
+    person("Evan L", "Ray", , "elray@umass.edu", role = "aut"),
+    person("Li", "Shandross", , "lshandross@umass.edu", role = c("aut", "cre"),
```

Comment:
While I accepted this suggestion at first, the R CMD checks fail without the extra comma. That comma seems to create an empty middle name field, and the emails are otherwise read as middle names with incorrect syntax. Is it ok to keep in the comma, or is it preferable to use a different solution?

Reply:
> While I accepted this suggestion at first, the R CMD checks fail without the extra comma. That comma seems to create an empty middle name field, and the emails are otherwise read as middle names with incorrect syntax. Is it ok to keep in the comma, or is it preferable to use a different solution?

Ah I was wondering why there was that extra comma. It's preferred to use named arguments. `person(given = "Li", family = "Shandross", email = "lshandross@umass.edu")`.